proposals/NNNN-swiftpm-per-target-swift-language-version-setting.md

```diff
@@ -0,0 +1,54 @@
+# Swift Language Version Per Target
+
+* Proposal: [SE-NNNN](NNNN-swiftpm-per-target-swift-language-version-setting.md)
+* Authors: [Pavel Yaskevich](https://github.com/xedin)
+* Review Manager: TBD
+* Status: **Awaiting Review**
+* Implementation: https://github.com/apple/swift-package-manager/pull/7439
+
```

Comment:
```suggestion
* Review: ([pitch](https://forums.swift.org/t/pitch-swiftpm-swift-language-version-per-target/71067))

```

Add the review header that links to forum discussions for pitch, review, etc.

Reply:
Done!